scripts/addr2line.py

```diff
@@ -131,7 +133,7 @@ def __call__(self, address: str):
             return " ".join([self._binary, address, '\n'])
         # We print a dummy 0x0 address after the address we are interested in
```

Comment:
This comment is outdated after the patch.

Reply:
fixed